<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_newWindow</class>
 <widget class="QDialog" name="Add_newWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>691</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_Sex">
      <property name="text">
       <string>Sex</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineEdit_Name"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_Number">
      <property name="text">
       <string>Number</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_Name">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_Address">
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_Name">
      <property name="text">
       <string>NameEnter</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_Sex">
      <property name="text">
       <string>SexEnter</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_Number">
      <property name="text">
       <string>NumEnter</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_Address">
      <property name="text">
       <string>AddrEnter</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEdit_Sex"/>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineEdit_Number"/>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="lineEdit_Address"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
